src/fairseq2/data/data_pipeline.py

```diff
@@ -115,6 +130,17 @@ def bucket_by_length(
         ) -> Self:
             """Combine examples of similar shape into batches."""
 
+        def collate(
+            self,
+            pad_idx: Optional[int] = None,
+            pad_to_multiple: int = 1,
+            overrides: Optional[Sequence["CollateOptionsOverride"]] = None,
+        ) -> Self:
+            """Concatenate a list of inputs into a single inputs.
+
+            See :py:class:`fairseq2.data.Collater` for details.
```

Comment:
Nit: Maybe also note that this is a convenience function over `map` + `Collater`?

Reply:
done